apps/gp2-frontend/src/outputs/OutputDetail.tsx

```diff
@@ -36,12 +38,15 @@ const OutputDetail: FC = () => {
   if (!output) {
     return <NotFoundPage />;
   }
-
   return (
     <Switch>
       <Route exact path={path}>
         <Frame title="Output">
-          <OutputDetailPage isAdministrator={isAdministrator} {...output} />
+          <OutputDetailPage
+            canVersion={isEnabled('ASAP_VERSION_RESEARCH_OUTPUT')}
```

Comment:
I believe there is a problem here, the button still appears even when the feature flag is false
![Screenshot 2023-11-20 at 08 42 00](https://github.com/yldio/asap-hub/assets/16595804/5bfb1262-f6ec-4acb-a3a6-e53a9fb908e6)

I believe it's not working properly because of this line
https://github.com/yldio/asap-hub/blob/master/packages/react-context/src/flags.tsx#L22

we remove `ASAP_` from the feature flag name, so maybe if you change to 

```suggestion
            canVersion={isEnabled('VERSION_RESEARCH_OUTPUT')}
```

it will work